ui/src/ui/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,179 @@
+.nav-main-container {
+  box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+  position: relative;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  z-index: 10000000 !important;
+  position: fixed;
+  width: 100%;
+}
+
+.logo {
+  width: 4.5rem;
+  background-color: #01645F;
+  padding: 0.5rem 0;
+}
+
+.nav-profile-image {
+  border-radius: 1000px;
+  width: 2.2rem;
+  cursor: pointer;
+  margin-right: 1.2rem;
+}
+
+.nav-profile-icon {
+  font-size: 1.75rem;
+  margin-right: 1.2rem;
+  cursor: pointer;
+}
+
+.nav-title {
+  text-transform: uppercase;
+  margin: 0 !important;
+  color: #D3D3D3;
+  font-weight: 500 !important;
+  font-size: 1rem;
+}
+
+.nav-profile-main-container {
+  position: fixed;
+  top: 0;
+  width: 100vw !important;
+  height: 100vh !important;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: 1;
+}
+
+.nav-profile-container {
+  position: fixed;
+  z-index: 2;
+  top: 2.9rem;
+  right: 0;
+  max-width: 17rem;
+  width: 100%;
+  padding: 2rem 0;
+  height: 100%;
+  background-color: white !important;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  /* overflow: hidden; */
+}
+
+.profile-image {
+  border-radius: 1000px;
+  width: 8rem;
+  padding: 0.2rem;
+  cursor: pointer;
+  border: 2px solid #00645F;
+}
+
+.profile-icon {
+  font-size: 11.5rem;
+  opacity: 20%;
+  border-radius: 1000px;
+  border: 1px solid black
+}
+
+.profile-displayName {
+  color: #888080;
+  font-weight: 600;
+  letter-spacing: 1px;
+  margin: 1rem 0 !important;
+  padding: 0 !important;
+  font-size: 1.1rem;
+  text-transform: capitalize;
+}
+
+.nav-info-button {
+  background: transparent;
+  text-transform: capitalize;
+  width: 89%;
+  display: flex;
+  align-items: center;
+  padding: 1rem;
+  gap: 1rem;
+  cursor: pointer;
+  margin-bottom: 0.4rem;
+}
+
+.nav-info-button-icon {
+  opacity: 60%;
+  font-size: 1.2rem;
+}
+
+.nav-info-hover {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-info-container {
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.3s ease;
+}
+
+.nav-info-button:hover {
+  background-color: #EBEBEB;
+}
+
+.nav-info-hover:hover .nav-info-container {
+  max-height: 5rem;
+}
+
+.nav-info-table td {
+  padding: 0.5rem;
+  font-size: 0.85rem ;
+}
+
+.nav-info-table-heading {
+  font-weight: 500;
+  background-color: #DBE9E8;
+  text-transform: capitalize;
+}
+
+.nav-info-table-content {
+  background-color: #F6F6F6;
+  width: 100%;
+}
+
+@media screen and (min-width: 1440px) {
+  .nav-title {
+    font-size: 1.8rem;
+  }
+
+  .logo {
+    width: 9rem;
+    padding: 0.7rem 0;
+  }
+
+  .nav-profile-image {
+    width: 3.8rem;
+    margin-right: 2rem;
+  }
+  
+  .nav-profile-icon {
+    font-size: 3.3rem;
+    margin-right: 2rem;
+  }
+}
+
+@media screen and (max-width: 675px) and (min-width: 450px) {
```

Comment:
Suggestion: You can also define the breakpoints(eg: phone, tablet, desktop) globally and than use them here